.github/workflows/build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install packages
-        run: yarn
+        run: yarn --frozen-lockfile
```

Comment:
Only concern with having this in all the files is the devex could be a bit annoying, but CI runs very quickly here…so I expect it's not worth intentionally omitting it from other workflows.